_StoreMixin.js

```diff
@@ -223,6 +224,26 @@ define([
 			return result;
 		},
 
+		refreshCell: function (cell) {
+			var row = cell.row;
+			var self = this;
+
+			return this.collection.get(row.id).then(function (item) {
```

Comment:
After using this on a project, I'm realizing it might make sense to make sure `this.collection` exists before we do this.

For that matter, I'm realizing it might make more sense to use `_renderedCollection`, but it might not actually matter (I'm pretty sure `get` is identical to the original collection anyway).